frontend/testing/requests/lesson.test.js

```
@@ -0,0 +1,579 @@
+                },
+              },
+            },
+          },
+        }),
+        cache: "no-store",
+      });
+      expect(result).toEqual(mockLesson);
+    });
+
+    it("returns undefined when no lesson is found", async () => {
+      const { fetchAPI } = require("@/lib/utils");
```

ESLint

[error] A `require()` style import is forbidden.

(@typescript-eslint/no-require-imports)